### PR TITLE
Filipino Translation - 1 107 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Translating Files : 
revision-selection.asc = 954 translated words
rewriting-history.asc = 153 translated words
Note : Code text and computer code text are not included as translated words.

### CODE REVIEW

File: book/07-git-tools/sections/revision-selection.asc [hunk 1]
```diff
@@ -180,11 +180,11 @@ Pagtatakbo ng `git show HEAD@{2.months.ago}` ay ipapakita sa iyo ang pagtutugma
 Kung ikaw ay mayroong UNIX o Linux na karanasan, maaari mong isipin na ang iyong reflog bilang bersyon ng Git sa kasaysayan ng shell, na nagpapahiwatig na kung ano ang naroroon ay malinaw na may kaugnayan lamang para sa iyo at iyong ``sesyon'', at walang kinalaman sa sinuman na maaaring nagtratrabaho sa parehong makina.
 ====
 
-==== Ancestry References
+==== Mga Reperensiya na Kanunununuan
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - add dash: `Kanunu-nunuan`
- change to `Mga Reperensiya ng Kanunu-nunuan`

File: book/07-git-tools/sections/revision-selection.asc [hunk 2]
```diff
@@ -180,11 +180,11 @@ Pagtatakbo ng `git show HEAD@{2.months.ago}` ay ipapakita sa iyo ang pagtutugma
...
 
-The other main way to specify a commit is via its ancestry.
-If you place a `^` (caret) at the end of a reference, Git resolves it to mean the parent of that commit.
-Suppose you look at the history of your project:
+Amg iba pang pangunahing paraan upang tiyakin ang commit ay dumaan sa kanang kanunununuan.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `Ang iba`
- `kanunu-nunuan`

File: book/07-git-tools/sections/revision-selection.asc [hunk 3]
```diff
@@ -180,11 +180,11 @@ Pagtatakbo ng `git show HEAD@{2.months.ago}` ay ipapakita sa iyo ang pagtutugma
...
+Kung ikaw ay maglagay ng `^` (caret) sa katupusan ng reperensiya, ang Git ay naglulutas at ito ay nangangahulugan na ang magulang sa commit na iyon.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `ang Git ay naglulutas nito upang mangahulugang magulang sa commit na iyon.`

File: book/07-git-tools/sections/revision-selection.asc [hunk 4]
```diff
@@ -213,10 +213,10 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
 ----
 
 [NOTE]
-.Escaping the caret on Windows
+.Pag-alis ng caret sa Windows
 ====
 
-On Windows in `cmd.exe`, `^` is a special character and needs to be treated differently. You can either double it or put the commit reference in quotes:
+Sa Windows sa `cmd.exe`, `^` ay isang espesyal na karakter at nangangailangan upang tratuhin ng naiiba. Maaari kang madoble ito o maglagay ng commit na reperensiya sa mga quote:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `Maaari mong doblehin`

File: book/07-git-tools/sections/revision-selection.asc [hunk 5]
```diff
@@ -248,11 +248,11 @@ Date:   Wed Dec 10 22:22:03 2008 +0000
     Some rdoc changes
 ----
 
-The other main ancestry specification is the `~` (tilde).
-This also refers to the first parent, so `HEAD~` and `HEAD^` are equivalent.
-The difference becomes apparent when you specify a number.
-`HEAD~2` means ``the first parent of the first parent,'' or ``the grandparent'' -- it traverses the first parents the number of times you specify.
-For example, in the history listed earlier, `HEAD~3` would be
+Ang iba pang pangunahing kanunununuan na pagtutukoy sa `~` (tilde).
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `kanunu-nunuan`

File: book/07-git-tools/sections/revision-selection.asc [hunk 6]
```diff
@@ -248,11 +248,11 @@ Date:   Wed Dec 10 22:22:03 2008 +0000
...
+Ito rin ang tumutukoy sa uang magulang, kaya ang `HEAD~` at `HEAD^` ay magkatumbas.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `unang magulang`

File: book/07-git-tools/sections/revision-selection.asc [hunk 7]
```diff
@@ -248,11 +248,11 @@ Date:   Wed Dec 10 22:22:03 2008 +0000
...
+Ang pagkakaiba ay nagiging maliwanag kapag ikaw ang tumukoy ng isang numero.
+Ang `HEAD~2` ay ibig sabihin ``ang unang magulang sa uang magulang,'' o ``ang lolo o lola'' -- ito ay dumadaan sa unang mga magulang sa bilang sa maraming naitukoy.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `ang unang magulang sa unang magulang`
- `` Ang `HEAD~2` ay nangangahulugang ``

File: book/07-git-tools/sections/revision-selection.asc [hunk 8]
```diff
@@ -264,7 +264,7 @@ Date:   Fri Nov 7 13:47:59 2008 -0500
     ignore *.gem
 ----
 
-This can also be written `HEAD^^^`, which again is the first parent of the first parent of the first parent:
+Maaari itong maisulat sa `HEAD^^^`, na muli ang unang magulang sa unang magulang sa unang magulang:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `Maaari itong maisulat na`

File: book/07-git-tools/sections/revision-selection.asc [hunk 9]
```diff
@@ -276,27 +276,27 @@ Date:   Fri Nov 7 13:47:59 2008 -0500
...
-You can also combine these syntaxes -- you can get the second parent of the previous reference (assuming it was a merge commit) by using `HEAD~3^2`, and so on.
+Maaari mo ring pagsamahin ang mga syntax na ito -- maaari kang makakuha ng pangalawang magulang sa nakaraang reperensiya (ipagpapalagay na ito ay na-merge commit) sa pamamagitan ng paggamit ng `HEAD~3^2`, at iba pa.
 
 [[_commit_ranges]]
-==== Commit Ranges
+==== Mga Lawak ng Commit
 
-Now that you can specify individual commits, let’s see how to specify ranges of commits.
-This is particularly useful for managing your branches -- if you have a lot of branches, you can use range specifications to answer questions such as, ``What work is on this branch that I haven’t yet merged into my main branch?''
+Ngayon na magagawa mo na ang pagtukoy sa indibidwal na mga commit, tingnan nati kung paano magtukoy ng mga lawak sa mga commit.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `tingnan natin`

File: book/07-git-tools/sections/revision-selection.asc [hunk 10]
```diff
@@ -276,27 +276,27 @@ Date:   Fri Nov 7 13:47:59 2008 -0500
...
-This basically asks Git to resolve a range of commits that are reachable from one commit but aren’t reachable from another.
-For example, say you have a commit history that looks like <<double_dot>>.
+Ang pinaka-karaniwang na lawak na pagtutukoy ay ang doubleng-dot na syntax.
+Ito talaga ay nagtatanong sa Git na malutas ang lawak ng mga commit na mapupuntahan mula sa isang commit ngunit hindi maabot mula sa kabila.
+Halimbawa, sabihin na mayroon kang isang kasaysayan ng Git na mukhang ganito <<double_dot>>.
 
 [[double_dot]]
-.Example history for range selection.
-image::images/double-dot.png[Example history for range selection.]
+.Halimbawa ng kasaysayan sa lawak ng pagpili.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: change `pagpili` to `pagpipilian`

File: book/07-git-tools/sections/revision-selection.asc [hunk 11]
```diff
@@ -276,27 +276,27 @@ Date:   Fri Nov 7 13:47:59 2008 -0500
...
+image::images/double-dot.png[Halimbawa sa kasaysayan para sa lawak ng pagpipilian.]
 
-Say you want to see what is in your `experiment` branch that hasn’t yet been merged into your `master` branch.
-You can ask Git to show you a log of just those commits with `master..experiment` -- that means ``all commits reachable from experiment that aren’t reachable from master.''
-For the sake of brevity and clarity in these examples, the letters of the commit objects from the diagram are used in place of the actual log output in the order that they would display:
+Sabihin na gusto mong makita ang kung ano ang nasa iyong `eksperimento` na branch na hindi pa pinagsama sa iyong `master` na branch.
+Maaari kang magtanong sa Git upang ipakita sa iyo ang log ng mga commit na may `master..experiment` -- ibig sabihin ``ang lahat ng mga commit ay mapupuntahan mula sa eksperimento na hindi pa naabot mula sa master.''
+Para sa kapakanan ng kaiklian at kalinawan sa mga halimbawang ito, ang mga litra sa commit na mga bagay mula sa diagram ay ginagamit sa lugar sa aktwal na log output sa pagkakasunod-sunod na sila ay magpakita:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `litra` to `letra`
- you could change `diagram` to `dayagram`

File: book/07-git-tools/sections/revision-selection.asc [hunk 12]
```diff
@@ -305,8 +305,8 @@ D
 C
 ----
 
-If, on the other hand, you want to see the opposite -- all commits in `master` that aren’t in `experiment` -- you can reverse the branch names.
-`experiment..master` shows you everything in `master` not reachable from `experiment`:
+Kung, sa kabila, ikaw ay gustong makita ang kabaligtaran -- lahat ng mga commit sa `master` na wala sa `experiment` -- maaari mong baligtarin ang mga pangalan ng branch.
+`experiment..master` ay nagpapakita sa iyo ng lahat sa `master` na hindi maabot mula sa `experiment`:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `` Ang `experiment..master` ``

File: book/07-git-tools/sections/revision-selection.asc [hunk 13]
```diff
@@ -315,24 +315,24 @@ F
...
+Kung ikaw ay magpatakbo ng `git push` ang iyong kasalukuyang branch ay nag-track sa `origin/master`, ang mga commit ay nailista sa pamamagitan ng `git log origin/master..HEAD` ay ang mga commit na iliipat sa server.
+Maaari mo ring iwanan ang isang bahagi ng syntax para magkaroon ng Git assume sa `HEAD`.
+Halimbawa, Maaari kang makakuha ng parehong mga resulta tulad ng sa nakaraang halimbawa sa pamamagitan nang pag-type ng `git log origin/master..` -- Git substitutes `HEAD` kung nawala ang isang panig.
 
-===== Multiple Points
+===== Maramihang mga Punto
 
-The double-dot syntax is useful as a shorthand, but perhaps you want to specify more than two branches to indicate your revision, such as seeing what commits are in any of several branches that aren’t in the branch you’re currently on.
+Ang dobleng-dot na syntax ay kapaki-pakinabang bilang isang shorthand, ngunit marahil gusto mong magtukoy ng higit dalawang mga branch upang ipahiwatig ang mga rebisyon, tulad ng nakikita kung ano ang mga commit na nasa ilang mga branch na wala sa branch na kasalukyang nasa iyo.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `magtukoy ng higit pa sa dalawang`

File: book/07-git-tools/sections/revision-selection.asc [hunk 14]
```diff
@@ -341,23 +341,23 @@ $ git log ^refA refB
...
+Halimbawa, kung gusto mong makita ang lahat ng mga commit na maaaring mapuntahan mula sa `refA` o `refB` ngunit hindi mula sa `refC`, maaari mong gamitin ang alinman sa:
 
 [source,console]
 ----
 $ git log refA refB ^refC
 $ git log refA refB --not refC
 ----
 
-This makes for a very powerful revision query system that should help you figure out what is in your branches.
+Ginagawa ito para sa isang napaka kapangyarihan na rebisyong query na sistema na dapat makatulong sa iyo na para malaman kung ano ang nasa iyong mga branch.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `napakamakapangyarihan`

File: book/07-git-tools/sections/revision-selection.asc [hunk 15]
```diff
@@ -341,23 +341,23 @@ $ git log ^refA refB
...
 
 [[_triple_dot]]
-===== Triple Dot
+===== Triple na Dot
 
-The last major range-selection syntax is the triple-dot syntax, which specifies all the commits that are reachable by _either_ of two references but not by both of them.
-Look back at the example commit history in <<double_dot>>.
-If you want to see what is in `master` or `experiment` but not any common references, you can run:
+Ang huling major na pagitan-pagpili na syntax ay ang tripleng-dot na syntax, na tumutukoy sa lahat ng mga commit na maaaring abutin sa _alinman_ sa dalawang mga reperensiya ngunit hindi sa pamamagitan ng dalawang magkapareho.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `major` to `pangunahing`
- you could translate `range-selection` to `pagpili ng hanay`

File: book/07-git-tools/sections/revision-selection.asc [hunk 16]
```diff
@@ -341,23 +341,23 @@ $ git log ^refA refB
...
+Tingnan uli ang halimbawa sa commit na kasaysayan sa <<double_dot>>.
+Kung gusto mong makita kung ano ang na sa `master` o `experiment` ngunit wala sa anumang karaniwang mga reperensiya, maaari kang magpatakbo:
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: - `kung ano ang nasa`
- `maaari kang magpatakbo ng:`

File: book/07-git-tools/sections/revision-selection.asc [hunk 17]
```diff
@@ -382,4 +382,4 @@ $ git log --left-right master...experiment
 > C
 ----
 
-With these tools, you can much more easily let Git know what commit or commits you want to inspect.
+Gamit ang mga kasangkapan, maaari mong mas madali mong maipaalam ni Get kung ano ang commit o mga commit na gusto mong siyasatin.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/revision-selection.asc]: `madaling maipaalam` remove the mong

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho. I have finished reviewing your work. Please see the comments.

File: book/07-git-tools/sections/revision-selection.asc (hunk 2 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 1 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 3 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 4 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 5 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 6 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 7 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 8 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 13 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 14 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 15 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 16 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

File: book/07-git-tools/sections/revision-selection.asc (hunk 17 above)
Comment (mind-blast) [reply book/07-git-tools/sections/revision-selection.asc]: done change

**Comment by mind-blast** [comment]
I have all changed as per suggestion. Thank you for the proof read.

**Pull request merged**